plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java

```diff
@@ -142,6 +148,20 @@ public void done(IJobChangeEvent event) {
       }
     });
     deploy.schedule();
+    
+    // https://wiki.eclipse.org/FAQ_How_do_I_find_the_active_workbench_page%3F
+    IWorkbenchWindow window = PlatformUI.getWorkbench().getActiveWorkbenchWindow();
```

Comment:
I've just updated that wiki page as this `PlatformUI.getWorkbench().getActiveWorkbenchWindow()` is actually an anti-pattern!

We have the `ExecutionEvent`, so just use:
```
IWorkbenchWindow window = HandlerUtil.getActiveWorkbenchWindow(event);
```